### DISCUSSION TITLE
What does the percentage mean when captions are enabled?

### DISCUSSION THREAD

**Discussion by shruggle** [extesy/hoverzoom#1008] [Q&A] [answered]
Apologies if this is not the right place to put this, not super familiar with GitHub etiquette. I just downloaded the extension and was trying some settings out, but I haven't been able to figure out what the percentage means that shows up above images when captions are enabled. I've attached an example.

thanks!
![image](https://user-images.githubusercontent.com/92273030/186063056-84835e64-4501-4567-96be-b772e3624e6c.png)

**Comment by extesy** [comment, marked as answer]
It's a scale - a relative size of the displayed image to the full image size. It's for cases when the original is very large and does not fit on the screen, then you'll see percentages less than 100.
